### DISCUSSION TITLE
I want the ability to choose the layout for my developer tools

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#69] [Wants]
I don't want the developer tools to automatically switch between being rows to columns based on their width—I want to be able to manually trigger that. Currently I keep my devtools at the width where they stay rows (since rows are better for my work flow, but sometimes I need more width).

I also want the ability to have the console drawer open as a column or a row. Currently all browsers have it split as a row.

Here are [some images for further clarification](https://imgur.com/a/AmDhCty).

<hr>

https://webwewant.fyi/wants/5f60115e2058780fd87faa72/

**Comment by mahmudzero** [comment]
submitted by me via form on website

**Comment by aarongustafson** [comment]
Paging @captainbrosset and @hxlnt.

**Comment by captainbrosset** [comment]
Thanks Aaron, I already sent this to Rachel some time ago, and I know there are plans underway related to letting people personalize their devtools more than currently possible.
Rachel will probably be able to expand more on this.

I think this is great feedback, and we should take it into account. It is far from the first time I hear people being frustrated by how DevTools re-arranges itself when you make it narrower or wider.
